src/ringmesh/geomodel/geomodel_mesh_entity.cpp

```diff
@@ -645,6 +645,12 @@ namespace RINGMesh
     }
 
     template < index_t DIMENSION >
+    index_t Corner< DIMENSION >::nb_mesh_elements() const
+    {
+        return 1;
```

Comment:
What is you create a Corner and then call nb_mesh_elements? You will get 1, but there is no vertex yet in the mesh. I think this return is wrong.

Reply:
We can define that a corner always has 1 mesh element but not necessarily 1 vertex (when the mesh is not initialized). In this case the mesh element is empty

Reply:
Maybe but not convinecd. Why the tests are failling?

Reply:
Saved geomodel3d.msh has a number (and order)  of Elements different from the template